### ISSUE TITLE
Completion list displaced when command palette open

### ISSUE THREAD

**Issue by ruza-net** [warpdotdev/warp#158]
### Describe the bug

```shell
When I open the command palette while having open a completion list, the completion list jumps to just above the palette.

It seems to be following the cursor - it jumps to the palette on the first cursor blink at then follows cursor location.
```

### To Reproduce

Steps to reproduce:
1. open a tab
2. type a command
3. hit 'Tab' to invoke completion list
4. hit '⌘P' to invoke command palette _(completion list jumps to it)_
5. type anything _(completion list follows the cursor)_

### Expected behaviour

```shell
The completion list should stay at its original text field and should not respond to events while the command palette is open.
```

### Screenshots

https://user-images.githubusercontent.com/6079523/131731712-9a6a0151-7181-4abd-874c-564dae3d4cdc.mov

### Operating System

MacOS

### OS Version

11.3

**Comment by charlespierce** [comment]
Hi @ruza-net, thanks for reporting! We'll take a look and get this fixed!

**Comment by ruza-net** [comment]
Issue seems to be resolved with v0.2021.09.08.20.40.stable_02
